File: 1227-number-of-equivalent-domino-pairs/1227-number-of-equivalent-domino-pairs.cs
public class Solution {
    public int NumEquivDominoPairs(int[][] dominoes) {
        var map = new int[100];
        int cnt = 0;
        foreach (var pair in dominoes) {
            int u = pair[0];
            int v = pair[1];
            int num = (u > v) ? v * 10 + u : u * 10 + v;
            cnt += map[num];

            map[num]++;
        }
        return cnt;
    }
}

File: 1227-number-of-equivalent-domino-pairs/1227-number-of-equivalent-domino-pairs.py
class Solution:
    def numEquivDominoPairs(self, dominoes: List[List[int]]) -> int:
        hash = [0] * 100
        cnt = 0
        for d in dominoes:
            u, v = d[0], d[1]
            num = v * 10 + u if u > v else u * 10 + v
            cnt += hash[num]
            hash[num] += 1
        
        return cnt